help/destinations/catalog/data-management/zetamarketingplatform.md

```diff
@@ -0,0 +1,144 @@
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>* To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+Below is an example of correct identity mapping when exporting profiles to [!DNL Zeta Marketing Platform].
+
+Selecting source fields:
+* Select an identity namespace (custom or standard such as `Email`) as source field that uniquely identifies a profile in Adobe Experience Platform and [!DNL Zeta Marketing Platform].
+* Select any XDM source profile attributes that need to be exported to and updated in the [!DNL Zeta Marketing Platform].
+
+Selecting target fields:
+* (Mandatory) Select `uid` as the target identity that is mapped to the source identity namespace. 
+* (Optional) Select `email_md5_id` as the target identity that is mapped to the source identity namespace that represents email md5 values. In this scenario, map `uid` to a source identity namespace.
+* Select any other destination profile attribute names that need to be mapped to corresponding XDM source profile attributes.
+
+![Identity mapping](../../assets/catalog/data-management-platform/zmp-extension/MappingExample.png)
+
+## Exported data / Validate data export {#exported-data}
+
+A successful audience syndication from experience platfrom to the ZMP will update the target customer list in the ZMP. The count as well as sample profiles in the target customer list will be exactly equal to the number of identities that were successfuly activated from experience platform as shown below.
+
+![Customer List in ZMP](../../assets/catalog/data-management-platform/zmp-extension/CustomerListInZMP.png)
+
+Each audience member that was syndicated from the experience platform will also be visible under **Audiences** > **People** in the ZMP. 
```

Comment:
Each audience member that was syndicated from the experience platform will also be visible under **Audiences** > **People** in the ZMP. 

Rephrase as:

Each audience member that was activated from Experience Platform will also be visible under **Audiences** > **People** in the ZMP.